lms/templates/static_htmlbook.html

```diff
@@ -93,20 +101,20 @@
     $(document).ready(function() {
         var options = {};
         %if 'url' in textbook:
-            options.url = "${textbook['url']}";
+            options.url = "${textbook['url'] | n, js_escaped_string}";
         %endif
         %if 'chapters' in textbook:
             var chptrs = [];
             %for chap in textbook['chapters']:
-                chptrs.push("${chap['url']}");
+                chptrs.push("${chap['url'] | n, js_escaped_string}");
             %endfor
             options.chapters = chptrs;
         %endif
         %if chapter is not None:
-            options.chapterNum = ${chapter};
+            options.chapterNum = ${int(chapter) | n, dump_js_escaped_json};
```

Comment:
Thanks Ned.  I'm going to go with what was here.  I appreciate the down vote on decode.utf8, and I was able to test.